ts/core/MmlTree/OperatorDictionary.ts

```diff
@@ -1222,8 +1242,8 @@ export const OPTABLE: {[form: string]: OperatorList} = {
     '\u2AD9': MO.REL,        // element of opening downwards
     '\u2ADA': MO.REL,        // pitchfork with tee top
     '\u2ADB': MO.REL,        // transversal intersection
-    '\u2ADC': MO.REL,        // forking
     '\u2ADD': MO.REL,        // nonforking
+    '\u2ADD\u0338': MO.REL,  // nonforking with slash
```

Comment:
So `2ADC` is supplanted by combining `2ADD` with a slash?
What if `2ADC` occurs in the input?

Reply:
It was remove from the table in Appendix C of the spec, so I removed it here.  I will add it back in at the bottom, with the extra additions.